src/client/communicator.go

```diff
@@ -64,41 +64,51 @@ func (this *Communicator) GetVmsData() (interface{}, error) {
 	if err != nil {
 		return nil, fmt.Errorf("getting vms data error: %s", err)
 	}
-	s, err := this.GetRegistryVms()
+	s, err := this.GetRegistryTemplatesData()
 	if err != nil {
-		return nil, fmt.Errorf("getting vms registry data error: %s", err)
+		return nil, fmt.Errorf("getting registry templates data error: %s", err)
 	}
+	templates := s.([]types.Template)
```

Comment:
This had to be changed to support GetRegistryTemplatesData returning interface{} instead of a string map